app/components/DataTable/components/CivicCellRenderer/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+
+import NewTabLink from '@/components/NewTabLink';
+import { ICellRendererParams } from '@ag-grid-community/core';
+
+type CivicCellRendererProps = ICellRendererParams['data'];
+
+const CivicCellRenderer = ({
+  data,
+}: CivicCellRendererProps): JSX.Element => {
+  const [link, setLink] = useState('');
+  const [text, setText] = useState('');
+
+  useEffect(() => {
+    if (data?.externalSource?.toLowerCase() === 'civic'
+      && typeof data?.externalStatementId === 'number') {
```

Comment:
would it help to coerce this into a number? or is this one of the fields that can be string as well

Reply:
Good catch. The API actually has this set as `Sq.TEXT` so I'll try using `parseInt` on it and verify with `isNan`